checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -23,9 +44,32 @@
 import java.util.Set;
 
 import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.ElementKind;
+import javax.lang.model.element.ExecutableElement;
 import javax.lang.model.element.VariableElement;
 
-import com.sun.source.tree.Tree;
+// TODO: Here is a list of nice-to-have features/tests but not critical to release the Lock Checker:
+
+// Add an enhancement so that implicit .toString() calls are recognized, i.e. when a non-String object
```

Comment:
Changed the comment to this (hopefully it's clearer):
// Add an enhancement so that implicit .toString() calls are de-sugared, e.g. "foo" + myVariable + "bar"
// is handled as "foo" + myVariable.toString() + "bar". This is important so that LockVisitor handles
// myVariable as the receiver parameter of a method call.